src/main/java/org/springframework/data/redis/connection/lettuce/LettuceConnectionFactory.java

```diff
@@ -735,6 +736,29 @@ public void setShutdownTimeout(long shutdownTimeout) {
 		getMutableConfiguration().setShutdownTimeout(Duration.ofMillis(shutdownTimeout));
 	}
 
+	/**
+	 * Returns the shutdown quiet period for shutting down the RedisClient (in milliseconds).
+	 *
+	 * @return shutdown timeout.
+	 * @since 2.2
+	 */
+	public long getShutdownQuietPeriod() {
```

Comment:
Let's not introduce property accessors for shutdown quiet period as this pattern is deprecated and we don't require these methods.